README.md

```diff
@@ -314,6 +316,28 @@ Consult the [Kroki documentation](https://kroki.io/#support) to find out which f
 | `kroki-fetch-diagram` | Define if we should download (and save on the disk) the images from the Kroki server.<br/>This feature is not available when running in the browser. | `false`
 | `kroki-http-method` | Define how we should get the image from the Kroki server. Possible values:<br/><ul><li>`get`: always use GET requests</li><li>`post`: always use POST requests</li><li>`adaptive`: use a POST request if the URI length is longer than 4096 characters, otherwise use a GET request</li></ul> | `adaptive` |
 
+### Default configuration
+
+By default, images are generated as svg when possible.
+To alter this, set `kroki-default-format` as an 'external', document header, or document body attribute:
+
+```adoc
+:kroki-default-format: png
+```
+
+'External' means of setting an attribute vary depending on your environment, but include CLI, API, and Antora playbook and component descriptor attribute settings.
```

Comment:
I think we should add this as a note/tip:

**ℹ️ NOTE:** An AsciiDoc attribute can be defined through the CLI or API, in the document’s header or in the document’s body. In addition, if you are using Antora, you can define AsciiDoc attributes in your playbook and/or in your component descriptor.

References:

- https://asciidoctor.org/docs/user-manual/#setting-attributes-on-a-document
- https://docs.antora.org/antora/2.3/page/attributes/#custom-attributes

I don't want to give the impression that this is something specific to this extension but rather that we are using a built-in feature (an AsciiDoc attribute).
I don't mind "duplicating" the documentation to explain where an attribute can be defined. 

Maybe we should add a concrete example? For instance: "it's possible to define the output format globally in an Antora documentation site if we define 	`:kroki-default-format: png` in the playbook."

Reply:
I made pretty much this change.  I don't understand what the ℹ️ character is, I just copied and pasted it, is it reasonably correct?

Reply:
Yes it's an hack to emulate an admonition using Markdown